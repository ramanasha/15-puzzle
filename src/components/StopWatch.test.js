import renderer from 'react-test-renderer';
import React from 'react';
import StopWatch from './StopWatch';

describe('StopWatch', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<StopWatch className="xyz" seconds={120} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
