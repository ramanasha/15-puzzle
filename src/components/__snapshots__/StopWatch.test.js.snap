exports[`StopWatch should render correctly 1`] = `
<div
  className="xyz fyeoun">
  Time elapsed: 
  120
  s
</div>
`;
